packages/create-atomic/templates/package.json

```diff
@@ -1,24 +1,33 @@
 {
   "name": "{{project}}",
-  "description": "Coveo Atomic Project",
+  "description": "{{titleCase project}} project",
   "version": "0.1.0",
   "scripts": {
     "postinstall": "npm run setup-server && npm run setup-cleanup",
     "setup-server": "node ./scripts/setup-server.js",
     "setup-cleanup": "node ./scripts/clean-up.js",
     "start-server": "node ./scripts/start-server.js",
-    "start-app": "node ./scripts/start-app.js",
```

Comment:
For a local dev setup, webpack does a great job and this script isn't needed anymore